.ci/scripts/install_windows.bat

```diff
@@ -1,6 +1,19 @@
-REM SPDX-FileCopyrightText: 2020 Intel Corporation
-REM
-REM SPDX-License-Identifier: MIT
+@echo off
```

Comment:
i don't think we can just replace existing MIT copyright with apache. it should consider initial license